server/tests/src/test/java/org/infinispan/server/functional/RestCacheManagerResource.java

```diff
@@ -0,0 +1,102 @@
+package org.infinispan.server.functional;
+
+import static org.infinispan.server.security.Common.sync;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import org.infinispan.client.rest.RestCacheManagerClient;
+import org.infinispan.client.rest.RestClient;
+import org.infinispan.client.rest.RestResponse;
+import org.infinispan.server.test.junit4.InfinispanServerRule;
+import org.infinispan.server.test.junit4.InfinispanServerTestMethodRule;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+/**
+ * @author Ryan Emerson
+ * @since 11.0
+ */
+public class RestCacheManagerResource {
+   @ClassRule
+   public static InfinispanServerRule SERVERS = ClusteredIT.SERVERS;
+
+   @Rule
+   public InfinispanServerTestMethodRule SERVER_TEST = new InfinispanServerTestMethodRule(SERVERS);
+
+   private final ObjectMapper mapper = new ObjectMapper();
+
+   @Test
+   public void testHealthStatus() throws Exception {
+      RestResponse restResponse = sync(client().healthStatus());
+      assertEquals(200, restResponse.getStatus());
+      assertEquals("HEALTHY", restResponse.getBody());
+   }
+
+   @Test
+   public void testHealthInfo() throws Exception {
```

Comment:
It seems it is already tested on https://github.com/infinispan/infinispan/blob/7470fe04cf105cf545d9ab7cf2889238de9a68d0/server/rest/src/test/java/org/infinispan/rest/resources/CacheManagerResourceTest.java#L82-L100